.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "2017-11/ConcurrencyLecture"]
```

Comment:
please remove the submodule if it's no longer needed

Reply:
apologies again! Removed.